package.json

```diff
@@ -14,90 +14,90 @@
-    "jimp": "^0.6.4",
-    "jugglingdb": "^2.0.1",
-    "koa": "^2.7.0",
+    "@azure/cosmos": "2.1.7",
+    "@bugsnag/js": "6.3.2",
+    "@ffmpeg-installer/ffmpeg": "1.0.19",
+    "@google-cloud/bigquery": "4.1.4",
+    "@google-cloud/firestore": "2.2.3",
+    "@sentry/node": "5.5.0",
+    "@tensorflow/tfjs-node": "1.2.3",
+    "@zeit/ncc": "0.17.0",
+    "analytics-node": "3.4.0-beta.1",
+    "apollo-server-express": "2.6.7",
+    "auth0": "2.18.0",
+    "axios": "0.19.0",
+    "azure-storage": "2.10.3",
+    "browserify-middleware": "8.1.1",
+    "canvas": "2.5.0",
+    "chromeless": "1.5.2",
+    "codecov": "3.1.0",
+    "consolidate": "0.15.1",
+    "copy": "0.3.2",
+    "cowsay": "1.4.0",
+    "esm": "3.2.25",
+    "express": "4.17.1",
+    "fetch-h2": "2.2.0",
+    "firebase": "6.2.4",
+    "firebase-admin": "8.2.0",
+    "fluent-ffmpeg": "2.1.2",
+    "graphql": "14.4.2",
+    "grpc": "1.21.1",
+    "highlights": "3.1.4",
+    "hot-shots": "6.3.0",
+    "ioredis": "4.11.1",
+    "isomorphic-unfetch": "3.0.0",
+    "jest": "24.5.0",
+    "jimp": "0.6.4",
+    "jugglingdb": "2.0.1",
+    "koa": "2.7.0",
     "leveldown": "4",
-    "lighthouse": "^5.1.0",
-    "loopback": "^3.26.0",
-    "mailgun": "^0.5.0",
-    "mariadb": "^2.0.5",
-    "memcached": "^2.2.2",
-    "mongoose": "^5.6.3",
-    "mysql": "^2.17.1",
-    "npm": "^6.10.0",
-    "oracledb": "^3.1.2",
-    "passport": "^0.4.0",
-    "passport-google-oauth": "^2.0.0",
-    "path-platform": "^0.11.15",
-    "pdf2json": "^1.1.8",
-    "pdfkit": "^0.10.0",
-    "pg": "^7.11.0",
-    "pug": "^2.0.4",
-    "react": "^16.8.6",
-    "react-dom": "^16.8.6",
-    "redis": "^2.8.0",
-    "rxjs": "^6.5.2",
-    "saslprep": "^1.0.3",
-    "sequelize": "^5.9.3",
-    "sharp": "^0.22.1",
-    "socket.io": "^2.2.0",
-    "socket.io-client": "^2.2.0",
-    "stripe": "^7.4.0",
-    "swig": "^1.4.2",
-    "tiny-json-http": "^7.1.2",
-    "twilio": "^3.33.0",
-    "vm2": "^3.8.2",
-    "vue": "^2.6.10",
-    "vue-server-renderer": "^2.6.10",
-    "when": "^3.7.8",
-    "yoga-layout": "^1.9.3"
+    "lighthouse": "5.1.0",
+    "loopback": "3.26.0",
+    "mailgun": "0.5.0",
+    "mariadb": "2.0.5",
+    "memcached": "2.2.2",
+    "mongoose": "5.6.3",
+    "mysql": "2.17.1",
+    "npm": "6.10.0",
+    "oracledb": "3.1.2",
+    "passport": "0.4.0",
+    "passport-google-oauth": "2.0.0",
+    "path-platform": "0.11.15",
+    "pdf2json": "1.1.8",
+    "pdfkit": "0.10.0",
+    "pg": "7.11.0",
+    "pug": "2.0.4",
+    "react": "16.8.6",
+    "react-dom": "16.8.6",
+    "redis": "2.8.0",
+    "rxjs": "6.5.2",
+    "saslprep": "1.0.3",
+    "sequelize": "5.9.3",
+    "sharp": "0.22.1",
+    "socket.io": "2.2.0",
+    "socket.io-client": "2.2.0",
+    "stripe": "7.4.0",
+    "swig": "1.4.2",
+    "tiny-json-http": "7.1.2",
+    "twilio": "3.33.0",
+    "typescript": "3.5.2",
+    "vm2": "3.8.2",
+    "vue": "2.6.10",
+    "vue-server-renderer": "2.6.10",
+    "when": "3.7.8",
+    "yoga-layout": "1.9.3"
```

Comment:
Let's not pin devDependencies though, the lock file handles this for us, and it's better to ensure the tests are updated I think, which will be painful if we pin here.